Adds (post): gentle introduction to distributed authorization in CanDIG

amanjeev opened the pull request:
@ljdursi This is a very simple introduction. Please review.

amanjeev committed:
Adds (post): gentle introduction to distributed authorization in CanDIG

amanjeev committed:
Fix (post): formatting

ljdursi commented on _posts/2020-03-04-gentle-intro-to-federated-distributed-authz.md:
```diff
@@ -0,0 +1,215 @@
+
+This model is an extremely simplified version of the Canadian healthcare system,
+which is governed exclusively by provincial jurisdiction, and each province has
+its privacy regulations. This is where CanDIG, as a platform, helps. It is a
```
I'd say more of the health research environment than the healthcare system in this case; maybe something like: "This is a simplified version of the Canadian health research environment: health is under provincial jurisdiction and each province has health privacy regulations."

ljdursi commented on _posts/2020-03-04-gentle-intro-to-federated-distributed-authz.md:
```diff
@@ -0,0 +1,215 @@
+- No central authorization authority
+- Authorization made locally based on local policies, informed by platform-wide CanDIG services
+- Site asked to provide data must be able to verify any such remotely provided information
+- Sites are completely symmetric from a Data Use & Access point of view
```
Can probably lose that last bullet point; Data Use & Access had a very specific meaning in the GA4GH plenary talk but might not mean anything here.

ljdursi reviewed (changes requested):
I think this is a very nice introduction.  Please take a look at the couple small tweaks I've proposed at the beginning, but otherwise this is great.

amanjeev committed:
Fix (post, gentle-intro-to-distributed-authz): health research

amanjeev committed:
Fix (post, gentle-intro-to-distributed-authx): removes unneeded line

Comment from Jonathan:
Can probably lose that last bullet point; Data Use &
Access had a very specific meaning in the GA4GH plenary
talk but might not mean anything here.

amanjeev commented:
@ljdursi thank you! I have made the changes.
